dubbo-common/src/main/java/org/apache/dubbo/common/extension/support/WrapperComparator.java

```diff
@@ -43,10 +43,8 @@ public int compare(Object o1, Object o2) {
             return 0;
         }
 
-        Class clazz1 = (Class) o1;
-        Class clazz2 = (Class) o2;
-
-        Class<?> inf = findSpi(clazz1);
```

Comment:
The result of the method call here is not used, so we can directly delete this line of code and the findSpi method below